sources/settings-migrations/v1.33.0/reset-metadata-migration/src/main.rs

```diff
@@ -0,0 +1,18 @@
+use migration_helpers::common_migrations::ResetMetadataMigration;
+use migration_helpers::{migrate, Result};
+use std::process;
+
+// Remove the weak settings on downgrade
+fn run() -> Result<()> {
+    migrate(ResetMetadataMigration)
+}
```

Comment:
nit: `ResetMetadataMigration` could be embedded here since it's unlikely we'll make use of it as a common migration

Reply:
Replaced this migration with RemoveMetadataAndWeakSettingsMigration.